s3_file_field/views.py

```diff
@@ -68,7 +67,12 @@ def upload_initialize(request: Request) -> HttpResponseBase:
     field = _registry.get_field(upload_request['field_id'])
 
     s3ff_upload_prefix = PurePosixPath(getattr(settings, 'S3FF_UPLOAD_PREFIX', ''))
```

Comment:
I propose we cut `S3FF_UPLOAD_PREFIX` entirely. This feature does not exist on the upstream `Storage` or `FileField`. Having it here just creates more complexity.

If downstreams want to add a prefix, they can do so by changing `upload_to`.